### PR TITLE
Fix/design of site

### PR DESCRIPTION
**Pull request by JacOng17**
Tweaked Menu, Disclaimer & Footer to match design.

### CODE REVIEW

**Comment by JacOng17** [comment]
image

**Comment by JacOng17** [comment]
image

**Comment by JacOng17** [comment]
image

File: _config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ baseurl: "/bagel-box"
 # the base hostname & protocol for your site, e.g. http://example.com
 url: "https://serviceinnovationlab.github.io/"
 
-github_username: ServiceInnovationLab
+github_username: Service Innovation Lab GitHub
```
Comment (mermop) [inline _config.yml]: Why this change?

Comment (mermop) [reply _config.yml]: You could just remove it from the config and change it in the footer

File: _includes/social.html [hunk 2]
```diff
@@ -2,7 +2,6 @@
   {%- if site.dribbble_username -%}<li><a href="https://dribbble.com/{{ site.dribbble_username| cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="{{ '/assets/minima-social-icons.svg#dribbble' | relative_url }}"></use></svg> <span class="username">{{ site.dribbble_username| escape }}</span></a></li>{%- endif -%}
```
Comment (mermop) [inline _includes/social.html]: Should we just remove this partial altogether

File: pages/intro.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 ---
-title: Machine Learning Primer
+title: Designing The Feedback Loop
```
Comment (mermop) [inline pages/intro.md]: Why did you change the title here? It is still the machine learning primer, not the designing the feedback loop page

Comment (mermop) [reply pages/intro.md]: You need to make another page for this

Comment (JacOng17) [reply pages/intro.md]: Oops, will fix that :+1:

File: _config.yml (hunk 1 above)
Comment (JacOng17) [reply _config.yml]: The design 
image

File: _sass/_disclaimer.scss [hunk 4]
```diff
@@ -1,16 +1,12 @@
 .disclaimer {
   width: 100%;
-  background-color: $color-bright;
+  height: 80%;
```
Comment (mermop) [inline _sass/_disclaimer.scss]: `height: 80%` is a bit of an odd decision - why are you defining this in % rather than in `em`s?

File: _sass/footer.scss [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  color: $color-lightest;
+  font-size: 13px;
+
+  a {
+    color: $color-lightest;
+  }
+}
+
+.username-github {
+  width: 208px;
```
Comment (mermop) [inline _sass/footer.scss]: This is a very oddly specific number - why?

File: _sass/footer.scss [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    .govt-logo {
+      height: 17px;
+      width: 170px;
+    }
+  }
+}
+
+.SIL-logo {
```
Comment (mermop) [inline _sass/footer.scss]: It's usually best to keep everything in lowercase

File: _sass/footer.scss [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.govt-logo {
+  height: 21px;
+  width: 200px;
+  padding: 0 20px 10px 0;
+}
+
+.cc-logo {
+  width: 85px;
```
Comment (mermop) [inline _sass/footer.scss]: Again, this is very specific... maybe you could achieve what you want by giving the parent element  `display: flex`

File: _sass/minima/_layout.scss [hunk 8]
```diff
@@ -12,37 +12,64 @@
...
 
-  .page-link {
-    color: $text-color;
-    line-height: $base-line-height;
+  .trigger {
+    a {
+      padding: 12px;
+      color: white;
+      text-decoration: none;
+      font-size: 17px;
```
Comment (mermop) [inline _sass/minima/_layout.scss]: Why 17px?

File: _sass/minima/_layout.scss [hunk 9]
```diff
@@ -12,37 +12,64 @@
...
+      font-weight: bold;;
+      text-align: center;
+    }
+
+    a:hover {
+      background-color: #DEDEDE;
```
Comment (mermop) [inline _sass/minima/_layout.scss]: Define this colour where the other colours are defined

File: _sass/minima/_layout.scss [hunk 10]
```diff
@@ -12,37 +12,64 @@
...
-  .page-link {
-    color: $text-color;
-    line-height: $base-line-height;
+  .trigger {
+    a {
+      padding: 12px;
+      color: white;
+      text-decoration: none;
+      font-size: 17px;
+      font-weight: bold;;
```
Comment (mermop) [inline _sass/minima/_layout.scss]:
```suggestion
      font-weight: bold;
```

File: _sass/minima/_layout.scss [hunk 11]
```diff
@@ -12,37 +12,64 @@
...
+    }
+    
+    .page-link {
+      color: #FFF;
+      line-height: $base-line-height;
 
-    // Gaps between nav items, but not on the last one
-    &:not(:last-child) {
-      margin-right: 20px;
+      // // Gaps between nav items, but not on the last one
```
Comment (mermop) [inline _sass/minima/_layout.scss]: Delete this if you're not using it - but why not use it?

**Comment by JacOng17** [comment]
image

**Review by mermop — APPROVED** [review]

File: _sass/footer.scss [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.govt-logo {
+  height: 21px;
+  width: 200px;
+  padding: 10px 20px 10px 0;
+}
+
+.username-github {
+  width: 208px;
```
Comment (mermop) [inline _sass/footer.scss]:
```suggestion
```

File: _sass/footer.scss [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+.sil-logo {
+  padding: 0 10px 0 0;
+
+  flex-shrink: 1;
+}
+
+.govt-logo {
+  height: 21px;
+  width: 200px;
```
Comment (mermop) [inline _sass/footer.scss]:
```suggestion
```

File: _sass/footer.scss [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.footer-logos {
+  display: flex;
+  width: 200px;
+}
+
+.sil-logo {
+  padding: 0 10px 0 0;
+
```
Comment (mermop) [inline _sass/footer.scss]:
```suggestion
width: 200px;
```

File: _sass/footer.scss [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+    .govt-logo {
+      height: 17px;
+      width: 170px;
+    }
+  }
+}
+
+.footer-logos {
+  display: flex;
+  width: 200px;
```
Comment (mermop) [inline _sass/footer.scss]:
```suggestion
  width: 100%;
```

File: pages/demo.md [hunk 16]
```diff
@@ -1,7 +1,8 @@
 ---
```
Comment (mermop) [inline pages/demo.md]: Delete this page
